TheMovieDB/TheMovieDBTests/MovieParserTest.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  MovieParserTest.swift
+//  TheMovieDBTests
+//
+//  Created by Mariana Andrea Orman Berch on 7/10/21.
+//
+
+import XCTest
+@testable import TheMovieDB
+
+class MovieParserTest: XCTestCase {
+
+    func movieParserTest() throws {
+
+        let fileUrl = URL(fileURLWithPath: "https://api.themoviedb.org/3/movie/top_rated?api_key=1f4d7de5836b788bdfd897c3e0d0a24b&language=en-US&page=1")
```

Comment:
Don't create a network call here because it will fail if there is no internet connection, instead download the JSON file and use it here